eo-maven-plugin/src/main/java/org/eolang/maven/OyRemote.java

```diff
@@ -81,4 +86,60 @@ public Input get(final String name) throws MalformedURLException {
         return new InputOf(url);
     }
 
+    /**
+     * Hash of head master.
+     * @param hash String "master"
+     * @return SHA of commit
+     * @checkstyle NonStaticMethodCheck (20 lines)
+     */
+    private String getSha(final String hash) {
+        String sha = "master";
+        try {
+            final String query = String.format(
+                // @checkstyle LineLength (1 line)
+                "https://api.github.com/repos/objectionary/home/git/refs/heads/%s",
+                hash
+            );
+            final JSONObject obj = OyRemote.readJsonFromUrl(query);
+            sha = obj.getJSONObject("object").getString("sha");
+        } catch (final IOException | JSONException exception) {
+            Logger.info(this, "Couldn't get commit SHA. It will be set as \"master\"");
```

Comment:
@yasamprom this is called "exception swallowing" -- it's a bad practice

Reply:
> @yasamprom this is called "exception swallowing" -- it's a bad practice

How should we process exceptions in this case? Is it better to stop program?

Reply:
@yegor256 I added logging exception

Reply:
@yasamprom it's always better to rethrow exceptions, instead of logging or swallowing